vslib/src/SwitchMLNX2700.cpp

```diff
@@ -48,6 +48,8 @@ sai_status_t SwitchMLNX2700::create_qos_queues_per_port(
         attr[1].id = SAI_QUEUE_ATTR_PORT;
         attr[1].value.oid = port_id;
 
+        // TODO add type
```

Comment:
Just want to confirm if I understood this correctly. Does this mean `SAI_QUEUE_ATTR_TYPE`?

Reply:
yes